sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -105,7 +105,7 @@
     <build.context>azure-client-sdk-parent</build.context>
     <jacoco.min.linecoverage>0.40</jacoco.min.linecoverage>
```

Comment:
Thoughts on updating this jacoco.min.linecoverage?

Reply:
Not sure if Jacoco has the ability to configure this with a command line property given how the configuration works.